packages/office-ui-fabric-react/src/components/Fabric/Fabric.base.tsx

```diff
@@ -87,7 +91,7 @@ export class FabricBase extends React.Component<
     }
   }
 
-  private _onMouseDown = (ev: MouseEvent): void => {
+  private _onPointerDown = (ev: PointerEvent): void => {
```

Comment:
I don't know if @ecraig12345 preferred using the `_onMouseDown` name that was already there, but since you're at least not using different methods this seems fine to me.

Reply:
Either is fine with me.

Reply:
Shouldn't this be both? Does onPointerDown get called when mouse down is fired?

Reply:
onPointerDown does get called when mouse down is fired.

Reply:
just checked, `onPointerDown` does fire on mouse down and all browsers support pointer down. we should remove `on(win, 'mousedown', this._onPointerDown, true),`

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onpointerdown

^According to that, Safari and Opera do not.

Reply:
so I'm a bit hesitant to remove that.

Reply:
indeed. i am not sure how i saw "all browsers support" :)

Reply:
Ah, this PR has been hanging for a while, we should close on this :)

re-reading the comments, I think here is what we need to do / what David was originally suggesting:
we should have 2 handlers - 
```
private _onMouseDown = (ev: MouseEvent): void => {
   this.setState({ isFocusVisible: false });
};

private _onPointerDown = (ev: PointerEvent): void => {
if (event.pointerType !== 'mouse') { // mouse type is handled by _onMouseDown
   this.setState({ isFocusVisible: false });
}
};
```